dipy/reconst/dki.py

```diff
@@ -0,0 +1,1634 @@
+            2) Three lines of the eigenvector matrix each containing the first,
+               second and third coordinates of the eigenvector
+            3) Fifteen elements of the kurtosis tensor
+
+    sphere : a Sphere class instance (optional)
+        If a sphere class instance is given, MK can be estimated faster as the
+        average of the directional kurtosis of the vertices in the sphere [1]_.
+        Otherwise MK is computed from its analytical solution [2]_.
+        
+    Returns
+    -------
+    mk : array
+        Calculated MK.
+
+    Notes
+    --------
+    The MK analytical solution is calculated using the following equation [2]_:
+
+    .. math::
+
+    \begin{multline}
+    MK=F_1(\lambda_1,\lambda_2,\lambda_3)\hat{W}_{1111}+
+       F_1(\lambda_2,\lambda_1,\lambda_3)\hat{W}_{2222}+
+       F_1(\lambda_3,\lambda_2,\lambda_1)\hat{W}_{3333}+ \\
+       F_2(\lambda_1,\lambda_2,\lambda_3)\hat{W}_{2233}+
+       F_2(\lambda_2,\lambda_1,\lambda_3)\hat{W}_{1133}+
+       F_2(\lambda_3,\lambda_2,\lambda_1)\hat{W}_{1122}
+    \end{multline}
+        
+    where $\hat{W}_{ijkl}$ are the components of the $W$ tensor in the
+    coordinates system defined by the eigenvectors of the diffusion tensor
+    $\mathbf{D}$ and 
+ 
+    \begin{multline}
+    F_1(\lambda_1,\lambda_2,\lambda_3)=
+    \frac{(\lambda_1+\lambda_2+\lambda_3)^2}
+    {18(\lambda_1-\lambda_2)(\lambda_1-\lambda_3)}
+    [\frac{\sqrt{\lambda_2\lambda_3}}{\lambda_1}
+    R_F(\frac{\lambda_1}{\lambda_2},\frac{\lambda_1}{\lambda_3},1)+\\
+    \frac{3\lambda_1^2-\lambda_1\lambda_2-\lambda_2\lambda_3-
+    \lambda_1\lambda_3}
+    {3\lambda_1 \sqrt{\lambda_2 \lambda_3}}
+    R_D(\frac{\lambda_1}{\lambda_2},\frac{\lambda_1}{\lambda_3},1)-1 ]
+    \end{multline}
+
+    \begin{multline}
+    F_2(\lambda_1,\lambda_2,\lambda_3)=
+    \frac{(\lambda_1+\lambda_2+\lambda_3)^2}
+    {3(\lambda_2-\lambda_3)^2}
+    [\frac{\lambda_2+\lambda_3}{\sqrt{\lambda_2\lambda_3}}
+    R_F(\frac{\lambda_1}{\lambda_2},\frac{\lambda_1}{\lambda_3},1)+\\
+    \frac{2\lambda_1-\lambda_2-\lambda_3}{3\sqrt{\lambda_2 \lambda_3}}
+    R_D(\frac{\lambda_1}{\lambda_2},\frac{\lambda_1}{\lambda_3},1)-2]
+    \end{multline}
+    
+    where $R_f$ and $R_d$ are the Carlson's elliptic integrals.
+      
+    References
+    ----------
+    .. [1] Hui ES, Cheung MM, Qi L, Wu EX, 2008. Towards better MR
+           characterization of neural tissues using directional diffusion
+           kurtosis analysis. Neuroimage 42(1): 122-34
+       [2] Tabesh, A., Jensen, J.H., Ardekani, B.A., Helpern, J.A., 2011.
+           Estimation of tensors and tensor-derived measures in diffusional
+           kurtosis imaging. Magn Reson Med. 65(3), 823-836
+    """
+    if sphere == None:
+        MK = _MK_analytical_solution(dki_params)
+    else:
+        MK = np.mean(apparent_kurtosis_coef(dki_params, sphere), axis=-1)
+
+    return MK
+
+
+def _MK_analytical_solution(dki_params):
```

Comment:
In python we try to not use capital letters in function names. But we use capital letters in classes.